.dockerignore

```diff
@@ -0,0 +1,4 @@
+dist/
```

Comment:
You are missing ``.git``

Reply:
Right. It doesn't have any impact on the final image, though. I'll add a fixup later